nannou/src/wgpu/texture/mod.rs

```diff
@@ -854,6 +854,20 @@ pub fn format_to_component_type(format: wgpu::TextureFormat) -> wgpu::TextureCom
         | wgpu::TextureFormat::Rgb10a2Unorm
         | wgpu::TextureFormat::Depth32Float
         | wgpu::TextureFormat::Depth24Plus
-        | wgpu::TextureFormat::Depth24PlusStencil8 => wgpu::TextureComponentType::Float,
+        | wgpu::TextureFormat::Depth24PlusStencil8
+        | wgpu::TextureFormat::Bc1RgbaUnorm
+        | wgpu::TextureFormat::Bc1RgbaUnormSrgb
+        | wgpu::TextureFormat::Bc2RgbaUnorm
+        | wgpu::TextureFormat::Bc2RgbaUnormSrgb
+        | wgpu::TextureFormat::Bc3RgbaUnorm
+        | wgpu::TextureFormat::Bc3RgbaUnormSrgb
+        | wgpu::TextureFormat::Bc4RUnorm
+        | wgpu::TextureFormat::Bc4RSnorm
+        | wgpu::TextureFormat::Bc5RgUnorm
+        | wgpu::TextureFormat::Bc5RgSnorm
+        | wgpu::TextureFormat::Bc6hRgbSfloat
+        | wgpu::TextureFormat::Bc6hRgbUfloat
+        | wgpu::TextureFormat::Bc7RgbaUnorm
+        | wgpu::TextureFormat::Bc7RgbaUnormSrgb => wgpu::TextureComponentType::Float,
```

Comment:
Now that https://github.com/gfx-rs/wgpu/pull/628 is merged, `format_to_component_type()` is no longer needed (we just can use `into()`).

Reply:
[8da1877 ](https://github.com/nannou-org/nannou/pull/655/commits/8da1877e80e364f04ad7d0dc6c899f39847c6354)